src/Universe.cpp

```diff
@@ -1420,18 +1506,19 @@ void Lattice::subdivideCells(double max_radius) {
 
   std::map<int, Universe*>::iterator iter;
   std::map<int, Universe*> universes = getUniqueUniverses();
-
-  /* Compute equivalent radius as the diagonal length to the cell boundary */
-  /* This is used as the maximum radius for all ringified Cells */
-  double radius = sqrt(_width_x*_width_x/4.0 + _width_y*_width_y/4.0);
-
-  /* If the lattice pitch is smaller than max_radius parameter, over-ride it */
-  if (radius < max_radius)
-    max_radius = radius;
+  
+  /* unique_radius is used as the maximum radius for the ringified Cells */
+  std::map<int, double> unique_radius = getUniqueRadius(universes);
 
   /* Subdivide all Cells */
-  for (iter = universes.begin(); iter != universes.end(); ++iter)
-    iter->second->subdivideCells(max_radius);
+  for (iter = universes.begin(); iter != universes.end(); ++iter) {
+    log_printf(DEBUG, "univ_ID: %d, radias: %f, max_radius: %f", 
+               iter->first, unique_radius[iter->first], max_radius);
+    
+    /* If the lattice pitch is smaller than max_radius parameter, over-ride it*/
```

Comment:
change to:
"If the local universe equivalent radius is smaller than the max radius, over-ride it"